public/app/features/alerting/unified/components/rule-editor/AnnotationsStep.tsx

```diff
@@ -105,9 +107,11 @@ const AnnotationsStep = () => {
     );
   }
 
+  const step = !isGrafanaManagedRuleByType(type) ? 5 : 6;
```

Comment:
Can we add a comment here to capture the intent / business logic associated here?